src/platform_impl/linux/window.rs

```diff
@@ -101,17 +107,21 @@ impl Window {
       .insert(window_id);
 
     // Set Width/Height & Resizable
-    let win_scale_factor = window.scale_factor();
     let (width, height) = attributes
       .inner_size
       .map(|size| size.to_logical::<f64>(win_scale_factor as f64).into())
       .unwrap_or((800, 600));
-    window.set_default_size(1, 1);
-    window.resize(width, height);
+
+    let window_clone = window.clone();
+    glib::idle_add_local_once(move || {
+      window_clone.set_resizable(true);
```

Comment:
why setting resizable to true `here`?  and we are not resetting it again at the end of the callback

Reply:
initially thought it would help create the accurate inner size. However, it was not useful, and I realized that the title actually affects the inner size. I’ll revert that change and then go to the header to adjust the title.